libobs/graphics/effect-parser.c

```diff
@@ -262,6 +307,144 @@ static void ep_parse_struct(struct effect_parser *ep)
 	ep_struct_free(&eps);
 }
 
+static inline int ep_parse_param_annotation_var(struct effect_parser *ep,
+	struct ep_param *var)
+{
+	int code;
+
+	/* -------------------------------------- */
+	/* variable type */
+
+	if (!cf_next_valid_token(&ep->cfp)) return PARSE_EOF;
+
+	if (cf_token_is(&ep->cfp, ";")) return PARSE_CONTINUE;
+	if (cf_token_is(&ep->cfp, ">")) return PARSE_BREAK;
+
+	code = cf_token_is_type(&ep->cfp, CFTOKEN_NAME, "type name", ";");
+	if (code != PARSE_SUCCESS)
+		return code;
+
+	bfree(var->type);
+	cf_copy_token(&ep->cfp, &var->type);
+
+	/* -------------------------------------- */
+	/* variable name */
+
+	if (!cf_next_valid_token(&ep->cfp)) return PARSE_EOF;
+
+	if (cf_token_is(&ep->cfp, ";")) {
+		cf_adderror_expecting(&ep->cfp, "variable name");
+		return PARSE_UNEXPECTED_CONTINUE;
+	}
+	if (cf_token_is(&ep->cfp, ">")) {
+		cf_adderror_expecting(&ep->cfp, "variable name");
+		return PARSE_UNEXPECTED_BREAK;
+	}
+
+	code = cf_token_is_type(&ep->cfp, CFTOKEN_NAME, "variable name",
+		";");
+	if (code != PARSE_SUCCESS)
+		return code;
+
+	bfree(var->name);
+	cf_copy_token(&ep->cfp, &var->name);
+
+	/* -------------------------------------- */
+	/* variable mapping if any (POSITION, TEXCOORD, etc) */
+
+	if (!cf_next_valid_token(&ep->cfp)) return PARSE_EOF;
```

Comment:
Style:  Avoid combining multiple statements on a single line.

Reply:
There were a few of these from the original code, wasn't sure if I should follow along w/ that at the time. I'll go over my functions again.

Reply:
There are some cases where I might consider it disruptive to stray from existing patterns within a particular file/subproject/submodule (e.g., alignments, indents, naming conventions).  This is not one of them.  I'm not saying you need to change all of the previously existing instances, just don't introduce new ones with that pattern.